Dockerfile

```diff
@@ -9,6 +9,8 @@ COPY ${SOURCE_CODE} .
 
 USER root
 
+RUN dnf install -y skopeo && dnf clean all
```

Comment:
Can we merge this line with line 14? It will be one less layer in the container image.